### ISSUE TITLE
Different output shape with padding="same" in Conv2DTranspose

### ISSUE THREAD

**Issue by hirokic5** [onnx/keras-onnx#555]
Hello, 

I use Conv2DTranspose in my model.  
But when converted it to onnx model,  the output of onnx model seems to differ from that of keras model.  
Maybe there is something wrong with padding... when padding="valid", the problem of output shape does not occur.  

Here is an example.
```
from tensorflow.keras.layers import Input, Conv2DTranspose, BatchNormalization, ReLU, Conv2D, Lambda, MaxPooling2D, Dropout,Lambda
from tensorflow.keras.layers import ZeroPadding2D
from tensorflow.keras.models import Model
from tensorflow.keras.regularizers import l2

def test_model(size=None,padding="same"):
    input_img = Input((size,size,3))
    x = Conv2DTranspose(256, (4, 4), strides=2, use_bias=False, padding=padding,
                        kernel_initializer='he_normal',
                        kernel_regularizer=l2(5e-4))(input_img)
    x = BatchNormalization()(x)
    y = ReLU()(x)
    return Model(inputs=input_img,outputs=y)

test = test_model(size=512)
print(test.input,test.output)

import keras2onnx
import numpy as np
import onnxruntime
test = test_model()
output_model_path = "weights/test.onnx"
onnx_model = keras2onnx.convert_keras(test, test.name)
keras2onnx.save_model(onnx_model, output_model_path)

x = np.zeros((512,512,3))
inputs = np.expand_dims(x,axis=0)
sess = onnxruntime.InferenceSession(output_model_path)
data = [inputs.astype(np.float32)]
input_names = sess.get_inputs()
feed = zip(sorted(i_.name for i_ in input_names), data)

actual = sess.run(None, dict(feed))

for k in actual:
    print(k.shape)
```
> tf executing eager_mode: True
tf.keras model eager_mode: False
The ONNX operator number change on the optimization: 12 -> 5
Tensor("input_5:0", shape=(None, 512, 512, 3), dtype=float32) Tensor("re_lu_4/Identity:0", shape=(None, 1024, 1024, 256), dtype=float32)
(1, 1026, 1026, 256)

thank you!

**Comment by jiafatom** [comment]
I just tried this model and it works fine, the output shape is `(1, 1024, 1024, 256)`. Can you pull latest keras2onnx master and try it again?

**Comment by hirokic5** [comment]
Thanks for reply!!  
I pulled latest keras2onnx(1.7.1,right?), but I got same odd shape...   

I refererd to [this notebook](https://github.com/onnx/keras-onnx/blob/master/tutorial/TensorFlow_Keras_EfficientNet.ipynb), and I have installed like below ; 

```
pip install --upgrade tensorflow==2.1.0
pip install onnxruntime
pip install git+https://github.com/microsoft/onnxconverter-common
pip install git+https://github.com/onnx/keras-onnx
```

**Comment by jiafatom** [comment]
We just added a unit test for your model. I tested your original model and it works fine (on our several configuration test). As a unit test, I made some simplification which does not affect your original problem.

**Comment by hirokic5** [comment]
Sorry for the late replay.  
I checked UT test, and tried again, but got odd shape...   

please would you tell your environment ?   

my environment : 
```
Anaconda on Windows10  
python : 3.7.2 
tensorflow : 2.1.0  
onnxruntime : 1.3.0  
keras2onnx : 1.7.1
```

**Comment by jiafatom** [comment]
Check our azure [pipelines](https://github.com/onnx/keras-onnx/tree/master/.azure-pipelines), each pipeline runs this test with several different environments. The build result can be seen [here](https://dev.azure.com/onnxmltools/ketone/_build)

**Comment by hirokic5** [comment]
Thanks, I'll check it !!

**Comment by hirokic5** [comment]
I found my problems in my code...  

when set size=None, the output shape is odd, but when set size=Constant(ex.128,256), the output shape is reasonable.  

Thanks for your replies and comments!!!
